sweep/README.md

```diff
@@ -0,0 +1,211 @@
+# Sweep
```

Comment:
Awesome doc! Really great read and super informative. I wonder if we should create a simple doc called `docs/sweep.md` that links to this one, so it can be found more easily? And we should definitely make sure this doc is linked in our online documentation (cc @Liongrass).

Reply:
yeah could do - will update this doc again in 18.1 so will add the `docs/sweep.md` then.